### ISSUE TITLE
Forbid single element destructing

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#1358]
# Rule request

## Thesis

This code should not be allowed:

```python
a, = [1]
(b,) = [1]
```

Related: https://github.com/d1618033/pylint-single-element-destructuring

## Reasoning

Why would you need to do it? Use `[0]` in this case. It is way more readable.

**Comment by nsidnev** [comment]
@sobolevn hi!
I'd like to take this. If I get it right, I have to change [this function](https://github.com/wemake-services/wemake-python-styleguide/blob/1edee56f7d7cd5cce537c4fe7dcd0175f8f04c50/wemake_python_styleguide/visitors/ast/builtins.py#L268) and add a new violation class there(`SingleElementDestructuring` as in `pylint` plugin for example), right?

**Comment by sobolevn** [comment]
@nsidnev thanks a lot!
